.gitignore

```diff
@@ -15,7 +15,10 @@
 *.out
 
 # Dependency directories (remove the comment below to include it)
-# vendor/
+vendor/
```

Comment:
It would be nice to include the CI actions, PR templates and dependabot config we have in most connectors, you can find them in the [template repo](https://github.com/ConduitIO/conduit-connector-template/tree/main/.github).

I would also like you to change the access settings of the repo and add @conduitio-labs/admins as the admin and @conduitio-labs/conduit-core as maintainers 🙏